data/nlu/nlu.md

```diff
@@ -4913,22 +4938,21 @@
 - thnks
 - thx
 - yes thanks
+- thanks for your information
 
-## lookup: location
-data/nlu/lookups/location.txt
```

Comment:
@ArjaanBuijk can you re-add the lookup tables on Rasa X? Not sure why these got deleted

Reply:
@akelad , I will look into this and fix it.